agent/handlers/v4/container_stats_handler.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We update our Copyright to a new format, can you change accordingly?

Reply:
sure, will change it